README.md

```diff
@@ -0,0 +1,69 @@
+# library-app
+[](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+## App
+
+![screenshot](./assets/menu.png)
+
+## About 
+
+This is a simple single-page application that displaces different content when a particular set of data is requested. It has three main sections.
+The home page which displaces details about a restaurant
+The menu page and,
+The contact page.
+
+## Built With
+
+- Javascript
+- Bootstrap
+- Html
+- Webpack
+
+
+### Prerequisites
+Knowledge about JS:
+- Basic data structures
+- Loops
+- Arrays
+- Functions
+- Constructors
+- Rendering HTML in JS file
+- JS injection
+- Knownledge of webpack
+
+## Setup
+- git clone:  [Github repo](https://github.com/nganifaith/library-app.git)
+- cd repo-folder OR
+- Open folder in your file explorer/finder
+- Start program by running live server Or
+- Click on the index.html file to open in your browser
+
+
+
+[Live site](https://nganifaith.github.io/library-app/)
```

Comment:
- [ ] The link you have added is for the library app. Please update it with the restaurant app.